deeprank/generate/DataGenerator.py

```diff
@@ -239,6 +242,7 @@ def create_database(
 
             # names of the molecule
             mol_name = os.path.splitext(os.path.basename(cplx))[0]
+            mol_name = mol_name.replace('-', '_')
```

Comment:
Why replacing `-` with `_` for molecule name?

Reply:
the dash causes issues in DeepXplorer and the simplest issue was to replace them with underscore

Reply:
I see, thanks